testers/sql/customize_test_env.sh

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env bash
 
-if [ $# -ne 2 ]; then
-    echo "Usage: $0 autotest_working_dir specs_dir"
+if [ $# -lt 2 ]; then
+    echo "Usage: $0 autotest_working_dir specs_dir" 
```

Comment:
Isn't venv_dir always passed now? Should be -ne 3 and have venv_dir in the echo.

Reply:
Yes the venv_dir is being passed but it is not required for this file (it's not used anywhere) so I though we should have a usage message that reflects how to run this file specifically, not how to run any customize_test_env.sh file in general.

Reply:
ouch it's the sql one, my bad